docs/docs/0.3.2/documentation/README.md

```diff
@@ -1841,8 +1847,40 @@ When `kuma-cp` starts up, by default it listens on a few ports:
 * `5682`: the HTTP server that provides the Envoy bootstrap configuration when the data-plane starts up.
 * `5683`: the HTTP server that exposes Kuma UI.
 
-## Quickstart
+### kuma-dp ports
+
+When we start a data-plane via `kuma-dp` we expect all the inbound and outbound service traffic to go through it. The inbound and outbound ports are defined in the [dataplane specification](#dataplane-specification) when running in universal mode, while on Kubernetes the service traffic always runs on port `15001`.
+
+In addition to the service traffic ports, the data-plane automatically also opens the `envoy` [administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) service on the following ports:
```

Comment:
> automatically also opens the `envoy`  administration interface *service*

when you call it a "service", it gives an impression that it should be possible to apply policies to it (which is not the case)